app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CartsController < ApplicationController
+  def update
+    @cart = Cart.find(params[:id])
+    if @cart.update_attributes(cart_params)
+      cart_params['cartitems_attributes'].each do |key, value|
```

Comment:
Unused block argument - `key`. If it's necessary, use `_` or `_key` as an argument name to indicate that it won't be used.